src/app/Playground.jsx

```diff
@@ -0,0 +1,260 @@
+// Framework tools
+import React from 'react';
+import { connect } from 'react-redux';
+import Radium from 'radium';
+
+// Playground-specific imports
+import Sidebar from 'components/playground/Sidebar';
+import Customizer from 'components/playground/Customizer';
+import Preview from 'components/playground/Preview';
+import Wizard from 'components/playground/Wizard';
+
+import DocumentTitle from 'react-document-title';
+
+import mediaQueries from 'playgroundSettings';
+
+import MdHelpOutline from 'react-icons/lib/md/help-outline';
+
+// Playground CSS
+require('../../css/playground.css');
+require('../../fonts/playground/coral-icon-font.css');
+
+@connect(state => state.tags)
+@Radium
+class Playground extends React.Component {
+
+  onWizardClick() {
+    this.setState({ wizardIsOpen: !this.state.wizardIsOpen });
+  }
+
+  hideWizard() {
+    this.setState({ wizardIsOpen: false }); 
+  }
+
+  render() {
+
+    var wizard = this.state.wizardIsOpen ? 
+      <div style={ styles.wizardOverlay } onClick={ this.hideWizard.bind(this) }>
+        <Wizard hideWizard={ this.hideWizard.bind(this) } />
+      </div>
+      : null;
+
+    return (
+      <DocumentTitle title="Coral Playground">
+
+        <div>
+
+          <div style={ [ styles.playgroundContainer, , this.state.wizardIsOpen ? styles.blurred : '' ] }>
+
+            <div style={ styles.playgroundLeftPane }>
+
+              <div style={ [ styles.playGround ] }>
+
+                <img src="http://coralproject.github.io/design/img/logos/coralWordMark-1.5.png" style={ styles.logo } />
```

Comment:
I'll add this image to the repo instead of hotlinking to another repo (that might change)